app/styles/components/canvas-glyph-input.scss

```diff
@@ -42,6 +42,10 @@
 				color: $theme-color;
 				background-color: $theme-background;
 				border-color: $theme-background;
+				transition-duration: .2s;
```

Comment:
Yup, that's expected. This way, the error color will never stay, it'll always be transitioning.
If I double the timeout value and an user keeps spamming an incorrect input, the red highlighting will be kinda shaky.